ti/callgraph/upload.go

```diff
@@ -26,11 +26,11 @@ const (
 )
 
 // Upload method uploads the callgraph.
-func Upload(ctx context.Context, stepID string, timeMs int64, out io.Writer) error {
+func Upload(ctx context.Context, stepID string, timeMs int64, out io.Writer, ticlient client.Client) error {
 	log := logrus.New()
 	log.Out = out
 
-	// TODO: Remove cyclic dependency
+	// TODO: Pass in the config here and use that, right now it will be empty
```

Comment:
What blocks to include ti config now? I thought we could retrieve the info from the saved mapping info?

Reply:
We are only passing in the client at the moment, we need to do the same for the TI Config (only used for RunTests step)